docs/src/reference/configuration.rst

```diff
@@ -141,455 +177,638 @@ title
+-  examples:
 
--  type: string
--  description: Path to `an Augur clade definition file <https://docs.nextstrain.org/en/latest/guides/bioinformatics/defining-clades.html#make-a-tsv-file-containing-your-clade-mutations>`__ for emerging lineages of concern that may be a subset or variation of the lineages defined by the ``clades`` parameter or Pangolin lineages.
--  default: ``defaults/emerging_lineages.tsv``
+   -  ``--max-date 2021-04-01``
+   -  ``--max-date 2021.25``
 
-filter
-------
+priorities
+~~~~~~~~~~
 
 -  type: object
--  description: Filters to apply to strain metadata and sequences prior to subsampling and tree inference. The workflow applies an implicit filter on the maximum collection dates later than today.
+-  description: Parameters to prioritize strains selected for the current subsampling rule. Currently, the workflow supports two ``type``\ s of priority, ``proximity`` and ``file``.
+-  description [proximity]: ``proximity`` selects samples that are genetically similar to the ``focus`` sample set; the ``focus`` sample set must be a rule in the current subsampling scheme.
+-  example [proximity]:
 
-.. contents::
-   :local:
+.. code:: yaml
 
-min_length
-~~~~~~~~~~
+   subsampling:
+     my-scheme:
+       my-first-rule:
+         max_sequences: 10
+       my-second-rule:
+         max_sequences: 10
+         # Prioritize sequences that are genetically similar to
+         # sequences in the sequences selected by the
+         # `my-first-rule` rule.
+         priorities:
+           type: proximity
+           focus: my-first-rule
 
--  type: integer
--  description: Minimum number of valid nucleotides (A, C, T, or G) for a genome to be included in the analysis by ``augur filter --min-length``.
--  default: ``27000``
+-  description [file]: ``file`` selects samples based on arbitrarily-defined rankings in a TSV file formatted as ``strain\tnumber``. The numbers are only used to sort the samples, and are therefore arbitrary. Higher values = higher priority.
 
-exclude_where
-~~~~~~~~~~~~~
+-  example [file]:
 
--  type: string
--  description: Conditional tests of metadata columns used to exclude strains from the analysis by ``augur filter --exclude-where``
--  default: ``"division='USA'"``
+.. code:: yaml
 
-exclude_ambiguous_dates_by
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+   subsampling:
+     my-scheme:
+       my-first-rule:
+         max_sequences: 10
+         group_by: "country"
+         priorities:
+           type: "file"
+           file: "path/to/priorities.tsv"
+
+::
+
+   hCoV-19/USA/CZB-1234/2021   8.2
+   hCoV-19/USA/CZB-2345/2021   0
+   hCoV-19/USA/CZB-3456/2021   -3.1
+
+
+Cross-Workflow configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These parameters are general high-level parameters which may affect multiple Snakemake rules, or modify which rules are run. 
```

Comment:
Two concerns:

1. I can understand the need to distinguish between **_Main Workflow Configuration_** and **_Cross-Workflow Configuration_**, but from a technical view **_Main Workflow Configuration_** also falls under the definition here.
2. With current descriptions, it isn't clear that `subsampling` is also a per-rule configuration.

Suggestion:

> **Primary configuration**
> 
> These parameters define the main inputs and outputs of the workflow, as well as the commonly used `subsampling` rule. Often these will be the only parameters you need to modify.
> 
> **Secondary configuration**
> 
> These parameters are other high-level parameters which may affect multiple Snakemake rules, or modify which rules are run.
> 
> **Per-Rule configuration**
> 
> Each top-level parameter here corresponds to a single Snakemake rule.
> `subsampling` is a commonly used rule configuration which is described separately in the **Primary configuration** section.

Reply:
Thanks for the improvements :) switched to these in [734bd66](https://github.com/nextstrain/ncov/pull/907/commits/734bd664f45071faee8769b8bb234403e206fd4e)